experiment/components/consent.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
```

Comment:
generic copy of the consent form from nivturk which felt robust and was sparse in autora-user-cookiecutter